modules/andes-core/broker/src/main/java/org/wso2/andes/store/rdbms/RDBMSAndesContextStoreImpl.java

```diff
@@ -832,6 +832,53 @@ public void deleteBindingInformation(String exchangeName, String boundQueueName)
         }
     }
 
+    @Override
+    public void updateExclusiveConsumerForQueue(String queueName, boolean isExclusiveConsumer) throws AndesException {
```

Comment:
GitHub has changed the mentioned comment. What I meant was to use "@inheritDoc" comment.